### PR TITLE
DOC-4225: New index page

### PR DESCRIPTION
**Pull request by beajohnson**

### CODE REVIEW

File: docs-src/zdm-core/modules/migrate/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration.
+
+A migration is a workflow that encompasses the lifecycle of uploadng and importing your data to the selected keyspaces. 
+At {company}, we've developed a set of thoroughly-tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration
+* Cassandra Data Migrations
+* DSBulk Migration
+
```
Comment (faizalrub-datastax) [inline docs-src/zdm-core/modules/migrate/pages/index.adoc:14]: The SSTable sideloader tool can also be included, depending on when this document is released. It is currently under private preview for certain customers. @jeromatron would be able to advise on when it will be available for GA. [SStable sideloader](https://docs.datastax.com/en/astra/astra-db-vector/sideloader/sideloader-overview.html). 
It might be appropriate to replace 'Cassandra Data Migrations' and 'DSBulk Migration' with 'Historic Data Migrations'. Cassandra Data Migrator and DSBulk are tools used to migrate historic data.

File: docs-src/zdm-core/modules/migrate/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+This means both systems remain synchronized without the need for downtime. 
+* Online database migration: Some database migration allows for the migration to happen while the database remains online and accessible. Technologies like database replication and synchronization tools can also facilitate seamless migration with minimal disruption.
+* Incremental data migration:Transfers data in smaller batches over time, which allows for minimal disruption to ongoing operations since only a portion of the data is being migrated at any given time. 
+This option can be used for large-scale migrations where downtime is not feasible or acceptable.
+* Load balancing and redundancy: In environments with redundant systems or load balancing capabilities, data migrations can be performed without downtime by routing traffic to the target system while the migration is in progress. This allows users to access services without interruption, as the load is distributed between the source and target systems.
+* Cloud-based data migration: This often provide tools and services that enable live data migration with minimal downtime. 
+
```
Comment (faizalrub-datastax) [inline docs-src/zdm-core/modules/migrate/pages/index.adoc:66]: The primary criterion for zero downtime migration is whether the business can tolerate a brief outage. Migration scenarios are documented here [Migration Scenarios](https://docs.datastax.com/en/astra/data-migration/introduction.html#migration-scenarios)

File: docs-src/zdm-core/modules/migrate/pages/index.adoc (hunk 1 above)
Comment (beajohnson) [reply docs-src/zdm-core/modules/migrate/pages/index.adoc:14]: The sideloader can only be seen in private preview. As soon as the release happens, I can add and publish this information.

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+image::{imagesprefix}zdm-migration-all.png[Migration phases from start to finish]
+
+Before walking through illustrations of each phase, look at a pre-migration, high-level view. At this point, your client applications are performing read/write operations with an existing CQL-compatible database. That is, Apache Cassandra, DSE, or Astra DB.  
+
+image::{imagesprefix}zdm-migration-before-starting.png[Diagram shows existing CQL-compatible environment before migration starts.]
+
+Before any migration begins, complete prerequisites, prepare your environment, and set up the recommended infrastructure.
+////
+
+== Migrations with and without zero downtime examples
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:45]: The phrasing of this heading is a little awkward and hard to parse. Please revise for clarity.

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Data migrations with downtime are those where the data being migrated is critical for ongoing operations, and the migration process itself requires systems to be taken offline temporarily. Downtime may be required for:
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:47]: Suggest breaking the first sentence into two shorter sentences and making the text more straightforward. For example, "Downtime may be necessary when you need to migrate data that is critical for ongoing operations. In some cases, the migration process itself requires systems to be taken offline temporarily."

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+* Migration for large-scale systems: these systems may require downtime for completely smooth transition.
+* Database migration: large database migrations or database software upgrades may need downtime for data consistency and integrity.
+* Storage system migration: migrating from on-prem to cloud may need downtime to reconfigure systems.
+* Network infrastructure changes: these systems may require downtime for completely smooth transition.
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:52]: The formatting of these bulleted items does not make sense with the intro sentence.  

Downtime may be required for: migration for large-scale systems: these...

Please revise the bulleted items so that they flow better with the intro sentence.

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+A migration with no downtime is seamless and ideal, but doesn't always happen. This happens when your business can tolerate a brief outage. 
+Here are scenarios for such migration:
+ 
+include::{imagesprefix}partial$migration-scenarios.adoc[]
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:57]: It's not clear how the text in migration-scenarious.adoc relates to no downtime.

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+== Data migration tools
+
+For modest data volumes where temporary downtime is tolerable, Datastax recommends two open-source data migration tools to help with brief outages:
+
+* Cassandra Data Migrator: The best choice to migrate large data quantities, and where detailed logging, data verifications, table column renaming (if needed), and reconciliation options are provided.
+* DSBulk Migrator: This leverages DataStax Bulk Loader (DSBulk) to perform the data migration, and provides new commands specific to migrations. DSBulk Migrator is ideal for simple migration of smaller data quantities, and where data validation (other than post-migration row counts) is not necessary.
+
+image::{imagesprefix}zdm-migration-phase2.png[Phase 2 diagram shows using tools to migrate data from Origin to Target.]
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:66]: I'm not sure why this image is here. It's below the bullet for DSBulk Migrator, but it's not clear that the image is related to that tool.

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+These tools provide sophisticated features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra, DSE, {company} {astra_db}).
+
+== Zero downtime migration
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:70]: ZDM is a type of data migration tool, so it should be included in the "Data migrations tools" section instead of its own section.

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+For large datasets, it's best to use the zero downtime migration tool: Zero Downtime Migration (ZDM).
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:72]:
```suggestion
For large datasets, it's best to use Zero Downtime Migration (ZDM).
```

File: modules/ROOT/pages/index.adoc [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+ZDM provides a simple and reliable way for you to migrate applications from any CQL-based cluster (Apache Cassandra®, {company} Enterprise (DSE), {astra_db}, or any type of CQL-based database) to any other CQL-based cluster, without any interruption of service to the client applications and data.
+
+* You can move your application to {astra_db}, DSE, or Cassandra with no downtime and with minimal configuration changes.
+* Your clusters will be kept in sync at all times by a dual-write logic configuration.
+* You can rollback at any point, for complete peace of mind.
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:77]: I'm not sure why these sentences are in bullets. Also, in the 3rd bullet, "rollback" should be two words (roll back) because it's being used as a verb.

File: modules/ROOT/pages/index.adoc [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+////
+Your migration project occurs through a sequence of phases. 
+Here is an example of these phases with ZDM using ZDM proxy, which is a simple and lightweight proxy that handles all the real-time requests generated by your client applications.
+
+image::{imagesprefix}zdm-migration-all.png[Migration phases from start to finish]
+////
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:84]: Remove this commented out section. This info should be in the intro to ZDM.

File: modules/ROOT/pages/index.adoc [hunk 12]
```diff
@@ -0,0 +1,86 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy,introduction
+ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration.
+
+A migration is a workflow that encompasses the lifecycle of uploadng and importing your data to the selected keyspaces.
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:8]:
```suggestion
A migration is a workflow that encompasses the lifecycle of uploading and importing your data to the selected keyspaces. 
```

**Review by KimberlyFields — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 7 above)
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc:66]: Faizal Rub suggested removing this image.

File: modules/ROOT/pages/index.adoc [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration.
+
+A migration is a workflow that encompasses the lifecycle of uploadng and importing your data to the selected keyspaces. 
+At {company}, we've developed a set of thoroughly-tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration
+* Cassandra Data Migrations
+* DSBulk Migration
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc:13]: I am planning to add links for all three when formatting for these docs is complete.

File: modules/ROOT/pages/index.adoc [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+For more, see http:addlinkhere.adoc[ZDM proxy]
+
+image::{imagesprefix}zdm-migration-all.png[Migration phases from start to finish]
+
+Before walking through illustrations of each phase, look at a pre-migration, high-level view. At this point, your client applications are performing read/write operations with an existing CQL-compatible database. That is, Apache Cassandra, DSE, or Astra DB.  
+
+image::{imagesprefix}zdm-migration-before-starting.png[Diagram shows existing CQL-compatible environment before migration starts.]
+
+Before any migration begins, complete prerequisites, prepare your environment, and set up the recommended infrastructure.
+////
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc]: I've removed it.

File: modules/ROOT/pages/index.adoc (hunk 5 above)
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc]: I've removed this content per Faizal. He said the main reason there really needs to be downtime is the size of the files. Large files need downtime.

File: modules/ROOT/pages/index.adoc [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+* Cassandra Data Migrator
+* DSBulk Migrator
+ 
+With these tools, you can move your application to {astra_db}, DSE, or Cassandra with no downtime and with minimal configuration changes.
+Your clusters are kept in sync at all times by a dual-write logic configuration.
+You can roll back at any point, for complete peace of mind.
+
+== Migration process
+
+Downtime may be necessary when you need to migrate data that is critical for ongoing operations.
```
Comment (faizalrub-datastax) [inline modules/ROOT/pages/index.adoc:21]: This should be worded as "If the business can tolerate some amount of application downtime and the data can be migrated within this period, tools such as Cassandra Data Migrator or DSBulk can be used for the migration."

File: modules/ROOT/pages/index.adoc [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+== Data migration tools
+
+For modest data volumes where temporary downtime is tolerable, Datastax recommends two open-source data migration tools to help with brief outages:
+
+* *Cassandra Data Migrator*: The best choice to migrate large data quantities, and where detailed logging, data verifications, table column renaming (if needed), and reconciliation options are provided.
+* *DSBulk Migrator*: This leverages DataStax Bulk Loader (DSBulk) to perform the data migration, and provides new commands specific to migrations. DSBulk Migrator is ideal for simple migration of smaller data quantities, and where data validation (other than post-migration row counts) is not necessary.
+
+These tools provide sophisticated features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra, DSE, {company} {astra_db}).
+
+For large datasets, it's best to use the Zero Downtime Migration (ZDM).
```
Comment (jeromatron) [inline modules/ROOT/pages/index.adoc:35]: ZDM is available for small and large data sets.  ZDM has two components
1. keeping the two databases in sync with ZDM proxy.  The proxy does the dual writing to both clusters.
2. data migration.  This step is necessary for any migration and we have CDM and dsbulk and soon Astra DB Sideloader when it GAs.  The data migration is part of the zero-downtime migration.  I'm not sure why we're separating ZDM from CDM/DSbulk.

File: modules/ROOT/pages/index.adoc (hunk 15 above)
Comment (alicel) [reply modules/ROOT/pages/index.adoc:21]: +1 to Faizal's suggestion

File: modules/ROOT/pages/index.adoc [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+== Migration process
+
+Downtime may be necessary when you need to migrate data that is critical for ongoing operations. 
+In some cases, the migration process itself may require systems to be taken offline temporarily. 
+Your business determines the downtime for any migration and how much downtime (if any) is necessary. 
+Large amounts of data require more time; small amounts of data require less time and may also be migrated through the {astra_ui}.
+
+== Data migration tools
+
+For modest data volumes where temporary downtime is tolerable, Datastax recommends two open-source data migration tools to help with brief outages:
```
[the remark this answers is not in the document]
Comment (alicel) [reply modules/ROOT/pages/index.adoc:28]: +1 to Faizal's suggestion, with a small wording change: `help with the migration of existing data` or `help migrating existing data`

File: modules/ROOT/pages/index.adoc (hunk 16 above)
Comment (alicel) [reply modules/ROOT/pages/index.adoc:35]: I do not understand the rationale for which we seem to recommend ZDM only for large datasets. ZDM can be used for migrations of any size and should be our go-to approach, while downtime migrations should be considered the exception.

File: modules/ROOT/pages/introduction.adoc [hunk 18]
```diff
@@ -113,71 +105,7 @@ Once that happens, the migration is complete.
...
-
-[.slide]
---
-.Phase 5: Connect directly to Target
-In this phase, move your client applications off the ZDM Proxy and connect the apps directly to Target. Once that happens, the migration is complete.
-
-image:migration-phase5ra9.png["Illustrates migration Phase 5, as summarized in the text. Back and Restart buttons are available for navigation within the graphic."]
---
-====
-////
```
Comment (alicel) [inline modules/ROOT/pages/introduction.adoc:180]: would this change remove the infographic?

Comment (beajohnson) [reply modules/ROOT/pages/introduction.adoc:180]: This has been removed.

File: modules/ROOT/pages/index.adoc [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+* *ZDM Proxy*: You can continue to run your current application and migrate data to the Origin database without any downtime. 
+It's a process of moving your current data from your application to the {company}'s Origin cluster seamlessly without stopping any connection to and from any cluster or application. 
+The proxy helps to manage the activity in transition.
+Your applications connect directly to the proxy instead of pointing to the database.
+NOTE: We recommend using three ZDM Proxies for this option. 
+* *Cassandra Data Migrator (CDM)*: This option migrates and validates data between Cassandra-compatible clusters. 
+It supports migration at scale and ensures accuracy and efficiency during the process.
+CDM allows for migration with zero downtime when used with the ZDM Proxy, and it can also be used for migrations with acceptable downtime. 
+* *DSBulk Loader*: This option is loading and unloading CSV/JSON data and counting table rows. 
+ DSBulk Loader does not inherently support zero downtime migrations like CDM when used with ZDM Proxy.
```
Comment (faizalrub-datastax) [inline modules/ROOT/pages/index.adoc:38]: DSBulk also allows for migration with zero downtime when used with the ZDM Proxy. It does not support data validation / reconciliation.

Comment (msmygit) [reply modules/ROOT/pages/index.adoc:38]: I'd make a slight adjustment to what Faizal had proposed above,
> It does not support data validation / reconciliation

to below,
> It does not support data validation / reconciliation other than validating the final table counts

File: modules/ROOT/pages/index.adoc [hunk 20]
```diff
@@ -23,18 +23,17 @@ When migrating data, you begin with an application connected to a cluster.
...
 It supports migration at scale and ensures accuracy and efficiency during the process.
 CDM allows for migration with zero downtime when used with the ZDM Proxy, and it can also be used for migrations with acceptable downtime. 
 * *DSBulk Loader*: This option is loading and unloading CSV/JSON data and counting table rows. 
- DSBulk Loader does not inherently support zero downtime migrations like CDM when used with ZDM Proxy.
+ DSBulk also allows for migration with zero downtime when used with the ZDM Proxy. It does not support data validation/reconciliation other than validating the final table counts.
 
-These tools provide sophisticated features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra, DSE, {company} {astra_db}).
+These tools provide features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra, DSE, {company} {astra_db}).
```
Comment (msmygit) [inline modules/ROOT/pages/index.adoc:39]: nitpick - in the 2nd appearance of `Apache Cassandra®` we're missing the ® symbol. Do we have a placeholder for this too in the antora yaml?

File: modules/ROOT/pages/index.adoc [hunk 21]
```diff
@@ -0,0 +1,39 @@
...
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration. 
+
+A migration is a workflow that encompasses the lifecycle of uploading and importing your data to the selected databases. 
+{company} has developed a set of thoroughly tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration (ZDM) Proxy
+* Cassandra Data Migrator
+* DSBulk Loader
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc:13]: I realize links should be added here. I wanted to have the other pages ready and live before adding the links. I can add them now.

File: modules/ROOT/pages/index.adoc [hunk 22]
```diff
@@ -0,0 +1,39 @@
...
+ 
+With these tools, you can move your application to {astra_db}, DSE, or Cassandra with no downtime and with minimal configuration changes.
+Your clusters are kept in sync at all times by a dual-write logic configuration.
+You can roll back at any point for complete peace of mind.
+
+== Migration process and tools
+
+{company} can migrate all data, however critical, with or without downtime from one application to our database.
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc:21]: The way Alice Lottini explained it, yes. The user is migrating data from their application to the Origin database.

File: modules/ROOT/pages/introduction.adoc (hunk 18 above)
Comment (jeromatron) [reply modules/ROOT/pages/introduction.adoc:180]: Is there a reason why we would remove the infographic?

**Review by jeromatron — COMMENTED** [review]
It looks like it adds a high level index to showcase the tools and then has a separate introduction document to introduce the reader to the zero-downtime migration process.  Is that accurate?  The only thing that I had a question on was the infographic - can we keep that in this doc flow?

File: modules/ROOT/pages/introduction.adoc [hunk 23]
```diff
@@ -62,8 +54,8 @@ The highlighted components in each phase emphasize how your client applications
 
 ==== Pre-migration client application operations
 
-Let's look at a pre-migration from a high-level view.
-At this point, your client applications are performing read/write operations with an existing CQL-compatible database: Apache Cassandra, DSE, or Astra DB.
+Here's a look at a pre-migration from a high-level view.
+At this point, your client applications are performing read/write operations with an existing CQL-compatible database: Apache Cassandra, DSE, or {astra_db}.
```
Comment (alicel) [inline modules/ROOT/pages/introduction.adoc:58]: I'd suggest changing the `:` to `such as`. This is a nuance, but we want to imply that there are other CQL-compatible databases that you may want to move from without mentioning them explicitly.

**Review by alicel — COMMENTED** [review]
The PR looks fine to me- there is just one sentence which requires some amendments, for which I left a comment.

Thank you!

File: modules/ROOT/pages/index.adoc [hunk 24]
```diff
@@ -0,0 +1,24 @@
...
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration. 
+
+{company} has developed a set of thoroughly tested self-service tools to walk you through well-defined migration options.
+These tools provide features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra®, DSE, {company} {astra_db}).
+
+== Migration process and tools
+
+A migration is a workflow that encompasses the lifecycle of uploading and importing your data to the selected databases. 
+{company} can migrate all data, however critical, with acceptable or zero downtime. 
+When the migration is complete, the data runs from your cluster to our database.
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/pages/index.adoc:15]: I replaced the current sentence with the following :
When the migration is complete, the data is present in the new database and all client applications connect exclusively to the new database. The old database becomes obsolete and can be removed.

**Review by alicel — APPROVED** [review]
Looks fine to me now. Thank you!

File: modules/ROOT/nav.adoc
Comment (KimberlyFields) [inline modules/ROOT/nav.adoc:1]: Bea, you have a different version of the migration nav file in every PR that you have. I suggest that you remove the nav file from this PR and update the nav in a single PR to avoid mistakes.

File: modules/ROOT/pages/index.adoc [hunk 25]
```diff
@@ -0,0 +1,24 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy, introduction
+ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration. 
+
+{company} has developed a set of thoroughly tested self-service tools to walk you through well-defined migration options.
+These tools provide features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra®, DSE, {company} {astra_db}).
```
Comment (KimberlyFields) [inline modules/ROOT/pages/index.adoc:9]: Use a lowercase o and t for origin and target.

**Review by KimberlyFields — APPROVED** [review]
